account_loan/wizard/account_loan_pay_amount.py

```diff
@@ -1,4 +1,3 @@
-# Copyright 2018 Creu Blanca
```

Comment:
why?

Reply:
Upps, I've no idea, a mistake removing encodings ? I revert it.